Ana_Mamani/practice09.py

```diff
@@ -0,0 +1,20 @@
+# Practice 07
```

Comment:
Please be careful with the language that you are using try to use all in English